subiquity/server/controllers/zdev.py

```diff
@@ -593,10 +596,43 @@
 id="0.0.c0fe" type="generic-ccw" on="no" exists="yes" pers="no" auto="no" failed="yes" names=""'''  # noqa: E501
 
 
+ZdevAiItem = TypedDict(
+    "ZdevAiItem",
+    {
+        "id": str,
+        "state": Literal["enabled", "disabled"],
```

Comment:
suggest using enable/disable here to match the apidef and to indicate that we are taking this action, not asserting initial state matches expectations.